lib/boardview.dart

```diff
@@ -300,22 +365,24 @@ class BoardViewState extends State<BoardView> with AutomaticKeepAliveClientMixin
 
   @override
   Widget build(BuildContext context) {
-    print("dy:${dy}");
-    print("topListY:${topListY}");
-    print("bottomListY:${bottomListY}");
-    if(boardViewController.hasClients) {
-      WidgetsBinding.instance!.addPostFrameCallback((Duration duration) {
+    super.build(context);
+
+    if (boardViewController.hasClients) {
+      WidgetsBinding.instance.addPostFrameCallback((Duration duration) {
         try {
           boardViewController.position.didUpdateScrollPositionBy(0);
-        }catch(e){}
-        bool _shown = boardViewController.position.maxScrollExtent!=0;
-        if(_shown != shown){
+        } catch (e) {}
+
+        bool _shown = boardViewController.position.maxScrollExtent != 0;
+
+        if (_shown != shown) {
           setState(() {
             shown = _shown;
           });
         }
       });
     }
+
     Widget listWidget = ListView.builder(
       physics: ClampingScrollPhysics(),
```

Comment:
Please add for ListView.builder customization 2 arguments Key? listViewKey and ScrollPhysics? listViewPhysics

```dart
Widget listWidget = ListView.builder(
      key: listViewKey,
      physics: listViewPhysics ?? const ClampingScrollPhysics() ,
      ...
```
I need these settings to control the offset across entire pages

Reply:
Hi @koutja kindly have this PR go in if what you've requested is a new feature it can be done on another PR, some of us depend on this. Thanks a lot.